README.md

```diff
@@ -142,7 +142,7 @@ See [Addon / Framework Support Table](https://storybook.js.org/docs/react/api/fr
 | [notes](https://github.com/storybookjs/deprecated-addons/tree/master/addons/notes) | Annotate Storybook stories with notes                      |
 | [options](https://www.npmjs.com/package/@storybook/addon-options)                  | Customize the Storybook UI in code                         |
 
-In order to continue improving your experience, we have to eventually deprecate certain addons in favor of new, better tools.
+In order to continue improving your experience, we have to eventually deprecate certain addons in favor of new and better tools.
```

Comment:
@virtualwiz1, we have a bit of room for improvement. We can go with "To continue improving your experience, we have to eventually deprecate certain addons in favor of new and better tools.".